xpublish/rest.py

```diff
@@ -172,7 +180,8 @@ def serve(self, host="0.0.0.0", port=9000, log_level="debug", **kwargs):
         -----
         This method is blocking and does not return.
         """
-
+        if self._app is None:
+            self.init_app()
```

Comment:
I think this is covered in the `@property` above.

Reply:
It is indeed. Thanks!